masschallenge/django-accelerator
# [AC-6621] Add sub navigation to the startup dashboard

### Changes introduced in [AC-6621](https://masschallenge.atlassian.net/browse/AC-6621):
- Add a migration to update the view profile and edit profile nav items with the profile nav item as their parent

## THIS HAS A SIBLING PR ON [ACCELERATE](https://github.com/masschallenge/accelerate/pull/2186) AND SHOULD BE MERGED BEFORE THAT IS MERGED

### How to test
- Once this migration visit the admin page for the [startup_dashboard_subnav](http://localhost:8181/admin/mc/navtree/7/change/), the Edit Profile and View Profile menu items should be nested under the Profile menu item
